src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -7,23 +7,22 @@ class InputEmptyBlocksAndCatchBlocks {
   public void fooMethod() {
     InputEmptyBlocksAndCatchBlocks r = new InputEmptyBlocksAndCatchBlocks();
     int a = 1;
-    if (a == 1) { } // violation 'Empty if block.'
+    if (a == 1) { } // no warn 'Empty if block.' until #15338
```

Comment:
I have keep `// no warn` comments only for `{ }` cases, I have removed them from all other places @rdiachenko

Reply:
all other places contained either K & R style code or `{}` code